Makefile.common

```diff
@@ -49,7 +49,13 @@ all: style staticcheck unused build test
 .PHONY: common-style
 common-style:
 	@echo ">> checking code style"
-	! $(GOFMT) -d $$(find . -path ./vendor -prune -o -name '*.go' -print) | grep '^'
+	@fmtRes=$$($(GOFMT) -d $$(find . -path ./vendor -prune -o -name '*.go' -print)); \
+	if [ -n "$${fmtRes}" ]; then \
+		echo "gofmt checking failed!"; echo "$${fmtRes}"; echo; \
+		echo "Note that the 'gofmt' output may vary depending on the go version."; \
```

Comment:
I'd embed the Go version into a sentence like: "Please ensure you are using Go version XXX for formatting code." so that people know how to fix this.